R/cite.R

```diff
@@ -84,7 +84,7 @@ cite <- function(type = c("plain", "bibtex"),
     "output.html"
   )
 
-  system2("pandoc", args)
+  system2(pandoc_path(), args)
```

Comment:
```suggestion
  system2(renv_pandoc_path(), args)
```

Reply:
but we should probably get rid of `cite()`, it was an internal utility that I tried putting together that doesn't really belong in renv.